boards/uvify/core/init/rc.board_sensors

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# UVify UVF4 specific board sensors init
+#------------------------------------------------------------------------------
+# Draco-R
+if param compare SYS_AUTOSTART 6002
+then
+   # Barometer
+   # Internal SPI
+   ms5611 -T 0 -s start
+
+   # PX4flow
+   px4flow start
+   ist8310 start
+   mpu6000 -R 2 -T 20608 start
+   mpu9250 -R 2 start
+   lis3mdl -R 2 -X start
+fi
+
+# Draco
+if param compare SYS_AUTOSTART 4052
+then
+   mpu9250 -R 2 start
```

Comment:
Why is the mpu9250 start dependent on the airframe?

Reply:
Because that one frame has mpu9250.